lgc/elfLinker/GlueShader.cpp

```diff
@@ -34,6 +34,7 @@
 #include "FetchShader.h"
 #include "NullFragmentShader.h"
 #include "lgc/state/PassManagerCache.h"
+#include "llvm/IR/LegacyPassManager.h"
```

Comment:
I understand that this code still uses the legacy pass manager because some upstream codegen passes have not been ported to the NPM yet.